modules/admin-ui/src/main/resources/public/org/opencastproject/adminui/languages/lang-de_DE.json

```diff
@@ -769,6 +769,8 @@
                "CAPTION": "Kommentare",
                "PLACEHOLDER": "Kommentieren…",
                "SELECTPLACEHOLDER": "Gründe wählen",
+               "NONE": "Keine",
+               "OPEN": "Offen",
```

Comment:
I feel like these are actually unnecessary since they are never used in the event details, unless I'm missing something?